src/include/ipc4/module.h

```diff
@@ -24,6 +24,8 @@
 #ifndef __IPC4_MODULE_H__
 #define __IPC4_MODULE_H__
 
+#include <ipc4/error_status.h>
+
```

Comment:
Unrelated?

Reply:
It is related. This header is now included in pipeline files from this patch and that breaks the build because this header uses macros, defined in error_status.h. This works before this patch because that header gets included via other paths currently, but since this header needs it, it should be included directly.